Sources/NFCPassportReader/PACEHandler.swift

```diff
@@ -171,6 +173,7 @@ public class PACEHandler {
     /// - Parameters:
     ///   - passportNonce: The decrypted nonce received from the passport
     func doStep2( passportNonce: [UInt8]) async throws -> OpaquePointer {
+        self.updateMessageHandler?(NFCViewDisplayMessage.paceAuthenticationPercentage(40))
```

Comment:
Can you elaborate a bit on that 40 and the 80 below? 🙏

Reply:
The PACE authentication is split into a 4-step process. It takes quite some time to complete the overall authentication, that's why I split it roughly 40-40-20, meaning step 1 takes approx. 40% of the overall PACE authentication duration, step 2 takes roughly the same amount of time (therefore, 80% of the authentication is complete by then), and steps 3 and 4 combined make up the remaining 20% of the time. 
Basically, these intermediate steps are just added to make the progress bar even smoother. If we only updated the progress bar before PACE starts and after it ends, it would take a bit too long to advance to the next progress dot.

Reply:
Sounds good.